.github/workflows/ui-test-vscuse.yml

```diff
@@ -271,6 +302,47 @@ jobs:
         with:
           ref: ${{ needs.setup.outputs.branch }}
 
+      - name: Ensure vscuse-related folder exists
```

Comment:
minor: I see this logic as well as Download vscuse-agent-cases artifact step repeated several times. May see how to share the logic if possible. Nice to have, can be improved later.

Reply:
Sure thanks.